Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -23,7 +25,18 @@ internal class InMemoryRepository<TItem> : IRepository<TItem> where TItem : clas
     {
         internal ConcurrentDictionary<string, string> Items { get; } = new();
 
-        internal string SerializeItem(TItem item) => JsonConvert.SerializeObject(item);
+        internal string SerializeItem(TItem item, string etag=null)
+        {
+            JObject jObject = JObject.FromObject(item);
+            if (etag != null)
+            {
+                jObject.Remove("_etag");
```

Comment:
If there is always an `"_etag"`, why not use `jObject.Replace` instead? Is there any benefit to `.Remove` and then `.Add`, or are they functionally equivalent?

Reply:
I have updated to use:
`jObject["_etag"] = JToken.FromObject(etag);`